tests/lib/rules/require-expect.js

```diff
@@ -294,6 +305,35 @@ ruleTester.run("require-expect", rule, {
             code: "test('name', function(assert) { other.expect(1); assert.ok(true); });",
             options: [],
             errors: [{ message: alwaysErrorMessage("assert.expect") }]
+        },
+
+        // "always" configuration - errors are reported on `test()` line
```

Comment:
Now that I know about using `column` I don't think this test adds value. Ok to remove?